### DISCUSSION TITLE
Oops，这里是 404 页面。

### DISCUSSION THREAD

**Discussion by akingzheng** [spencerwooo/onedrive-vercel-index#808] [Q&A]
Oops，这里是 404 页面。
{"error":{"code":"504","message":"An error occurred with your deployment"}}
大神们帮忙看一下为什么无数据呢
https://github.com/akingzheng/onedrive-vercel
https://onedrive.mjin.top/
![数据库](https://user-images.githubusercontent.com/20454916/207493120-309e79b6-9cde-4cd5-a521-9d21444329b2.jpg)
![无数据](https://user-images.githubusercontent.com/20454916/207493213-c2e84365-766d-4a56-a90e-22dba1fa6d58.jpg)

**Comment by spencerwooo** [comment]
在你附的这张图里：

![](https://user-images.githubusercontent.com/20454916/207493120-309e79b6-9cde-4cd5-a521-9d21444329b2.jpg)

`REDIS_URL` 应该是有问题的。这个链接我这里类似这样：

![CleanShot 2022-12-14 at 11 33 52](https://user-images.githubusercontent.com/32114380/207499073-0fbde462-efd6-46fb-bd98-21dddff08e60.png)

特别注意是，如果你开了 TLS，这个链接应该是 `rediss://xxx` 开头的，总之检查这个链接有效性。

（问题应该并不出现在 `default` 上，应该是 Upstash 后续更新之后加了这个值，但是我这里还没有更新。）
